tests/test_web_response.py

```diff
@@ -1171,3 +1173,36 @@ def test_text_is_json_encoded(self) -> None:
     def test_content_type_is_overrideable(self) -> None:
         resp = json_response({"foo": 42}, content_type="application/vnd.json+api")
         assert "application/vnd.json+api" == resp.content_type
+
+
+@pytest.mark.skipif(
+    not bool(os.environ.get("PYTHONASYNCIODEBUG")),
+    reason="Only run the test with debug mode",
```

Comment:
Why? Is it possible to enable the debug mode for this test temporarily?

Reply:
`sys.flags.dev_mode` is read-only, so not that I can see...

Reply:
Unless anybody has any other ideas, then I think we'll just need to add an extra pytest invocation to CI for dev mode as I mentioned before, using something like `python -X dev -m pytest`.

Reply:
While flags are provisioned at the beginning of the interpreter startup, I was under impression that it's the loop mode that needed to be set.

Reply:
Or we could replace the test invocation with `python -X dev -m pytest` instead of re-running all the tests in debug mode.

Reply:
I think that may be expensive. I'd rather not slow down the whole test run. Why does this need to depend on the global debug, anyway?

Reply:
> Or we could replace the test invocation with `python -X dev -m pytest` instead of re-running all the tests in debug mode.

Yes, sorry, back to my original suggestion. Replacing the `skipif` with a `pytest.mark.dev_mode`, and then we can run `python -X dev -m pytest -m dev_mode`. This will only run the dev_mode tests in the extra invocation.

Reply:
And we then need to disable tests marked as dev_mode by default for the other runs.

Reply:
> Why does this need to depend on the global debug, anyway?

https://github.com/aio-libs/aiohttp/blob/master/aiohttp/helpers.py#L75